akka-bbb-apps/src/main/scala/org/bigbluebutton/core/running/MeetingActor.scala

```diff
@@ -365,7 +374,9 @@ class MeetingActor(
   private def handleMessageThatAffectsInactivity(msg: BbbCommonEnvCoreMsg): Unit = {
 
     msg.core match {
-      case m: EndMeetingSysCmdMsg     => handleEndMeeting(m, state)
+      case m: EndMeetingSysCmdMsg =>
+        meetingEndTime = System.currentTimeMillis()
```

Comment:
In some cases you set `meetingEndTime` inside the handler method while in others you do it prior to calling the handler. Would you consider always doing it in the handler?

Reply:
I would have done it in this case as well, but in this scenario the `EndMeetingSysCmdMsgHdlr` does not actually have access to the fields in the `MeetingActor` so it cannot modify the `meetingEndTime`.